### PR TITLE
Remove $ in bash commands

### PR DESCRIPTION
**Pull request by QRemy**
Follow-up of #5113.

### CODE REVIEW

**Review by bkhelifi — APPROVED** [review]
Thanks for the effort ;)

**Review by Astro-Kirsty — COMMENTED** [review]
Thanks @QRemy!
Did you build the docs to ensure everything looks okay too? 
If so, all good on my side

File: docs/user-guide/scripts/index.rst [hunk 1]
```diff
@@ -121,15 +121,15 @@ a datastore and producing the reduced datasets.)
...
     run     Performs automated data reduction process.
 
-    $ gammapy analysis config
+    gammapy analysis config
     INFO:gammapy.scripts.analysis:Configuration file produced: config.yaml
 
 
 You can manually edit this produced configuration file and the run the data reduction process::
 
 .. code-block:: text
```
Comment (Astro-Kirsty) [inline docs/user-guide/scripts/index.rst:130]:
```suggestion
You can manually edit this produced configuration file and the run the data reduction process:

.. code-block:: bash
```

**Comment by Astro-Kirsty** [comment]
Built the docs and fixed the small error, ready for merge

**Review by Astro-Kirsty — APPROVED** [review]

**Comment by QRemy** [comment]
Thanks @Astro-Kirsty, merging now.

**Pull request merged**